### PR TITLE
Remove Last Gdi32 Structs/Enums to Use Cswin32

### PR DESCRIPTION
**Pull request by lonitra**
- Remove structs: `BITMAP`,`BITMAPINFOHEADER`,`LOGBRUSH`,`LOGPALETTE`,`LOGPEN`,`RGNHEADER`,`TEXTMETRICW`,`RGBQUAD`
- Remove enums: `BI`,`BS`,`EMR`,`ICM`,`MWT`,`OUT_PRECIS`,`PS`,`QUALITY`,`RGN`,`TMPF`,`StockObject`
- Move enum `FW` to Gdi folder

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/winforms/pull/7857)

### CODE REVIEW

File: src/System.Windows.Forms.Primitives/src/System.Windows.Forms.Primitives.csproj [hunk 1]
```diff
@@ -21,6 +21,12 @@
     <ProduceReferenceAssembly>true</ProduceReferenceAssembly>
     <UsePublicApiAnalyzers>true</UsePublicApiAnalyzers>
   </PropertyGroup>
+  
+  <ItemGroup>
```
Comment (lonitra) [inline src/System.Windows.Forms.Primitives/src/System.Windows.Forms.Primitives.csproj:25]: Not sure if this needs to be in here? This code was added when I deleted the Gdi32 folder.

**Review by JeremyKuhne — APPROVED** [review]

File: src/System.Windows.Forms.Primitives/src/System.Windows.Forms.Primitives.csproj (hunk 1 above)
Comment (Tanya-Solyanik) [reply src/System.Windows.Forms.Primitives/src/System.Windows.Forms.Primitives.csproj:25]: Once the GDI folder is removed, we don't need to exclude it in the project file. This code is not needed

**Review by RussKie — COMMENTED** [review]
LGTM modulo the changes to the csproj

**Review by Tanya-Solyanik — APPROVED** [review]
Looks good!

**Pull request merged**